baybe/utils/dataframe.py

```diff
@@ -21,6 +22,8 @@
     from baybe.targets.base import Target
 
     _T = TypeVar("_T", bound=Parameter | Target)
+    _ArrayLike = TypeVar("_ArrayLike", np.ndarray, Tensor)
```

Comment:
why not https://numpy.org/devdocs/reference/typing.html#numpy.typing.ArrayLike instead of the ndarray?

Reply:
ok ty for clarifying